geoapi/src/main/python/README.md

```diff
@@ -1,43 +1,95 @@
-# GeoAPI Python REAME
+# GeoAPI Python README
 
 ## Static Type Checking
-Static type checking for GeoAPI's Pytnon implementation is accomplished using
-[mypy](https://mypy.readthedocs.io/en/stable/index.html).
+Static type checking for GeoAPI's Pytnon implementation is accomplished using [mypy](https://mypy.readthedocs.io/en/stable/index.html).
```

Comment:
We can keep this on two lines in the `.md` file. They will still appear as a single paragraph in the rendered page. When using Markdown, all consecutive lines are in the same paragraph. A real line feed is done only when there is a blank line in the `.md` file.

The reason for splitting on many lines is to make the `.md` file easier to read, but also for smaller diff in commits. For better change isolation in commit, I often split the lines at natural boundaries such as a coma or a period.